isodesign/base/calculation.py

```diff
@@ -194,58 +200,98 @@ def __init__(self):
+        logger.debug(f"Prefixes from netw files in the folder = {self.netw_files_prefixes}")
 
-        # namedtuple containing file path and data 
-        files_infos = namedtuple("file_info", ['path', 'data'])
-        if ext in self.FILES_EXTENSION:
-            data = pd.read_csv(str(data_path), sep="\t", comment="#", header=None if ext == ".netw" else 'infer')
-            self.imported_files_dict.update({ext[1:]: files_infos(data_path, data)})
+    def load_file(self, prefix):
+        """ 
+        Load tvar, mflux, miso, cnstr and netw files (csv or tsv)
+        depending on the prefix chosen.
 
-            self.prefix = data_path.stem
-        
+        :param prefix: str containing the prefix of the file to be loaded
 
-    def initialize_data(self, directory):
         """
-        Imports all files in a folder
+        self.prefix = prefix
+        logger.info(f"Prefix {self.prefix}")
 
-        :param directory: folder path 
-        """
-        self.path_input_folder = Path(directory)
-        logger.debug(f"Input folder directory = {self.path_input_folder}")
+        if prefix not in self.netw_files_prefixes:
+            msg = f"Prefix '{prefix}' is not found in this folder."
+            logger.error(msg)
+            raise ValueError(msg)
+    
+        # namedtuple containing file path and data 
+        files_infos = namedtuple("file_info", ['path', 'data'])
+        for files in self.path_input_folder.iterdir():
+            if files.stem == prefix and files.suffix in self.FILES_EXTENSION:
+                data = pd.read_csv(str(files), sep="\t", comment="#", header=None if files.suffix == ".netw" else 'infer')
+                self.imported_files_dict.update({files.suffix[1:]: files_infos(files, data)})
 
-        for file in self.path_input_folder.iterdir():
-            self.read_file(str(file))
-        logger.debug(f"Prefix {self.prefix}")
         logger.debug(f"Imported files dictionary = {self.imported_files_dict}")
 
+
+    def input_network_analysis(self):
+        """
+        Analyzes the input network using modules from influx_si 
+        """
+        
+        # Reset self.netan to a new empty dictionary
+        # Useful if you want to reuse the function for another prefix
+        self.netan = {}
+        # temporarily stores files generated by the use of modules
+        with tempfile.TemporaryDirectory() as tmpdir:
+            for contents in self.path_input_folder.iterdir():
+                # copy all files contained in self.path_isodesign_folder (contains input files, linp files and vmtf file) 
+                if contents.is_file():
+                    shutil.copy(contents, tmpdir)
+
+            # will contain the paths to the ftbl files generated by the txt2ftbl module
+            li_ftbl = []  
+            # convert mtf to ftbl
+            txt2ftbl.main(["--prefix", os.path.join(str(tmpdir), self.prefix)], li_ftbl) 
+            # parse and analyze first ftbl store in li_ftbl
```

Comment:
"Parse and analyze ftbl" tout simplement car on en créé qu'un seul normalement si je ne m'abuse

Reply:
Si, c'est plusieurs ftbl qui doivent être créés, un par mélange de labeling.

Reply:
D'accord merci pour la précision :)